diff --git a/node_modules/@internationalized/number/dist/main.js b/node_modules/@internationalized/number/dist/main.js
index 8b8facd..7ca3272 100644
--- a/node_modules/@internationalized/number/dist/main.js
+++ b/node_modules/@internationalized/number/dist/main.js
@@ -48,6 +48,11 @@ const $b6bf6c42eb93d35942f2d0c9c4a9aa06$var$UNITS = {
       // https://bugs.webkit.org/show_bug.cgi?id=218139
 
     }
+  },
+  celsius: {
+    narrow: {
+      default: '°C'
+    }
   }
 };
 
